From 027f75cb7919a24820780492604eec79eef8b2ca Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 16 Jan 2023 20:56:32 +0200
Subject: [PATCH] [SW-99614] gaudi2: security: remove curly brackets

No need for curly brackets for one liners - remove those.
While at this, wrap the long line of the function call.

Change-Id: I889fdde50f6ad783fabb45b51acf8128bd326fe9
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi2/common/security.c b/arch/gaudi2/common/security.c
index 99f38be..ee595cc 100644
--- a/arch/gaudi2/common/security.c
+++ b/arch/gaudi2/common/security.c
@@ -324,9 +324,9 @@
 		}
 	}
 
-	if (rr_sel.pcie) {
-		hl_secure_lbw_rr_gen(rr_idx, min, max, 0, HL_SEC_PCIE_INST_NUM, HL_SEC_ENGINE_PCIE);
-	}
+	if (rr_sel.pcie)
+		hl_secure_lbw_rr_gen(rr_idx, min, max, 0, HL_SEC_PCIE_INST_NUM,
+					HL_SEC_ENGINE_PCIE);
 
 	if (rr_sel.psoc)
 		hl_secure_hbw_rr_gen(HL_SEC_HBW_PSOC_LBW, min, max, 0, HL_SEC_PSOC_INST_NUM,
